### ISSUE TITLE
[Feature] Editor support for multiple matching rules

### ISSUE THREAD

**Issue by xianath** [ankit/stylebot#674]
### Is your feature request related to a problem? Please describe.
I use Stylebot for an intranet system used by many different teams. I have a global site-wide style, and page-specific styles for team workspaces. Stylebot correctly matches and applies all rules, however the editor lacks an option to edit a particular rule, forcing me to use the Options screen and refresh, which is cumbersome and error-prone.

### Describe the solution you'd like
When matching multiple URLs, I'd like to have an option to edit the stylesheet for a particular rule. This could be done, for example, by adding a drop-down to select the stylesheet to edit. Another option is to display, in the editor itself, the sum of all applied stylesheets, with color coding or some other visual indication as to the rule they originate from, ideally also with the ability to easily move CSS styles between Stylebot rules.

### Additional context
Mock-up

![image](https://user-images.githubusercontent.com/14057138/140712688-260abb74-5f1d-4735-b267-91e39d68a153.png)

**Comment by ankit** [comment]
@xianath makes sense - An earlier version used to support this. I will try and bring this back.
